docs/src/examples/scikit-learn.rst

```diff
@@ -5,8 +5,8 @@ Scikit-learn
    repository. We invoke this file from index.rst
 
 In this example, we're going to demonstrate how Oríon can be integrated to a minimal model using
-scikit-learn [scikit]_ on the iris dataset [iris]_. The files mentioned in this example are
-available in the folder :file:`examples/scikitlearn-iris/`.
+`scikit-learn <https://scikit-learn.org/stable/index.html>`_ on the `iris dataset <https://en.wikipedia.org/wiki/Iris_flower_data_set>`_. The files mentioned in this example are
```

Comment:
For iris I would give this [link](http://archive.ics.uci.edu/ml/datasets/Iris) instead so that user are just one click away from downloading it.